modules/apps/27-interchain-accounts/controller/keeper/keeper_test.go

```diff
@@ -85,16 +85,16 @@ func SetupICAPath(path *ibctesting.Path, owner string) error {
 	return nil
 }
 
-// InitInterchainAccount is a helper function for starting the channel handshake
-func InitInterchainAccount(endpoint *ibctesting.Endpoint, owner string) error {
+// RegisterInterchainAccount is a helper function for starting the channel handshake
+func RegisterInterchainAccount(endpoint *ibctesting.Endpoint, owner string) error {
```

Comment:
Small nit: I was thinking we could maybe rename this test helper func to `initialiseHandshake` since there's some tests where it's called directly without continuing the handshake with `endpoint.ChanOpenTry()` or `endpoint.ChanOpenAck()`..etc being called after.

Feel free to leave as is, it can always be done later.